Alt text - almanac chapter

AlbaMenendezPereda opened the pull request:
Added alt text in figures.yaml to Figures 1-3 (Almanac)

AlbaMenendezPereda committed:
Alt text - almanac chapter

Added alt text in figures.yaml to Figures 1-3 (Almanac)

geealbers commented on content/_data/figures.yaml:
```diff
@@ -3,20 +3,23 @@ figure_list:
   - id: "fig-001"
     src: "figures/fig-001.jpg"
     zoom: true
+    alt: "Title page of a French almanac that features writing in French that identifies the title of the almanac, the publication year, the contents, and the name of the publisher. The page includes a coat of arms topped by a crown and embraced by two angel figures, one on each side of the crest."
```
The wording in the first part feels very repetitive:

> Title page of a French almanac that features writing in French that identifies the title of the almanac ...

The words "title", "French", and "almanac" are all used twice. It makes sense why it's this way, but for someone listening to this description read aloud, I imagine the repetitions might actually make it confusing. Can you reword?

geealbers commented:
@AlbaMenendezPereda I commented on one of them, but you can proceed with the other chapters and just continue committing them to this same branch. We'll merge them in when we reach some sort of critical mass.

AlbaMenendezPereda committed:
Alt text reviews and additions

Review of alt text for figures on almanac and addition of alt text to figures on Armchair, Baptism Certificate, and Bed. No figures in Bath

AlbaMenendezPereda committed:
Alt Text Added

Alt text added to the following sections: book, burin, camera obscura, carriage, color box, crayon, and decoration

AlbaMenendezPereda committed:
revisions to alt text Figs 1-17

AlbaMenendezPereda committed:
Alt Text Figs 18-64

AlbaMenendezPereda committed:
enter alt text to Figures 65 - 97

AlbaMenendezPereda committed:
enter alt text Figures 98 - 107

AlbaMenendezPereda committed:
enter alt text Figures 107 - 136

AlbaMenendezPereda committed:
enter alt text Figures 137-191

final batch

AlbaMenendezPereda commented:
@geealbers FYI: I just committed the last batch of alt text captions

geealbers committed:
Move extra alt entry to its correct fig

geealbers reviewed (approved):
Thanks for writing all of these @AlbaMenendezPereda!!

There was one small file issue where you had two alt texts on fig. 103 and none on fig. 102. (I found it by running the figures.yaml file through https://www.yamllint.com/ to check for formatting issues before previewing the site.) One of the extras on 103 was the one you wrote for 102, so I fixed that quickly and pushed a new commit. Otherwise, this all looks good and I'm going to go ahead and merge it in.

geealbers commented:
Actually, I'm not going to merge it into `main` just yet. There were a couple other fixes I want to make that weren't related to the alt text. I'll just make them on this branch.

AlbaMenendezPereda commented:
Sorry about that and thank you for spotting it!

geealbers committed:
Pull select accessibility updates from newer version of Quire

geealbers committed:
Add empty alt attributes for decorative images

geealbers committed:
Ensure lists in thing page tombstones have commas in PDF

geealbers committed:
Update output files

geealbers committed:
Update revisions statement
